mobly/controllers/attenuator.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3.4
+#
+# Copyright 2016 Google Inc.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib
+import logging
+
+MOBLY_CONTROLLER_CONFIG_NAME = "Attenuator"
+KEY_ADDRESS = "Address"
+KEY_PORT = "Port"
+
+
+def create(configs):
+    objs = []
+    for c in configs:
+        attn_model = c["Model"]
+        # Default to telnet.
+        protocol = c.get("Protocol", "telnet")
+        # Import the correct driver module for the attenuator device
```

Comment:
Not sure why we need to do this import; won't register_controller(minicircuits) take care of this?